src/batch/parquet-schema.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Parquet Schemas for DNSP Message Types
+ * See DSNP type definitions in DSNP.d.ts for additional documentation of fields.
+ */
+
+/**
+ * Broadcast: a public post
+ */
+export const BroadcastSchema = {
+  actionType: { type: "INT32" },
+  fromAddress: { type: "BYTE_ARRAY" },
+  messageID: { type: "BYTE_ARRAY" },
+  uri: { type: "BYTE_ARRAY" },
+};
+
+export const ReplySchema = {
```

Comment:
the other schemas have comments explaining what they are for. Do we want a comment for reply schema and drop schema below?

Reply:
Yes, I added them.